package/MDAnalysis/analysis/hydrogenbonds/wbridge_analysis.py

```diff
@@ -1210,18 +1210,24 @@ def _donor2acceptor(self, donors, h_donors, acceptor):
                                            box=self.box,
                                            return_distances=True)
         if self.distance_type != 'heavy':
+            tmp_distances = distances
             tmp_donors = [h_donors[donors_idx[idx]] for idx in pairs[:, 0]]
             tmp_hydrogens = [donors_idx[idx] for idx in pairs[:, 0]]
             tmp_acceptors = [acceptor[idx] for idx in pairs[:, 1]]
         else:
+            # To make sure that for the same index i, the donor (tmp_donors[i]),
```

Comment:
The `else` should be `elif self.distance_type == 'heavy':` followed by an `else: raise ValueError` (and a removal of the comment in `__init__` on distance_type.

Reply:
With the test in `__init__`, this is ok now.

Reply:
@orbeckst Thanks for the review. Given that I have already a check in __init__, I think it might not be good to do this any more as this is checked several times through out the code and it would be nice if they all use the same if statement. Instead of having `if self.distance_type != 'heavy':` in line 1205 and if `self.distance_type == 'hydrogen':` in line 1216